splice_machine/src/test/java/com/splicemachine/triggers/Trigger_Exec_Stored_Proc_IT.java

```diff
@@ -416,20 +416,20 @@ public void testStatementTriggerSysStoredProc() throws Exception {
     @Test @Ignore("DB-3424: Executing trigger action insert thru a stored proc (disallowed directly) causes infinite recursion.")
     public void testExecSQLRecursion() throws Exception {
 
-        methodWatcher.executeUpdate("insert into t_master values (13, 'grrr', 'I''m a pirate!', CURRENT_TIMESTAMP)");
-        methodWatcher.executeUpdate("insert into t_slave values (99, 'trigger01', CURRENT_TIMESTAMP)");
-        methodWatcher.executeUpdate("insert into t_slave2 values (22, 'trigger02', CURRENT_TIMESTAMP)");
+        methodWatcher.executeUpdate("insert into t_first values (13, 'grrr', 'I''m a pirate!', CURRENT_TIMESTAMP)");
+        methodWatcher.executeUpdate("insert into t_second values (99, 'trigger01', CURRENT_TIMESTAMP)");
+        methodWatcher.executeUpdate("insert into t_second2 values (22, 'trigger02', CURRENT_TIMESTAMP)");
```

Comment:
The "second2" naming is a bit confusing, I'd like more primary/secondary/secondary2, source/sink/sink2, source/target/target2 ...